Sources/HaCWebsiteLib/Events/Event.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+protocol Event {
+    var title : String {get}
+    var time : Date {get}
+    var eventDescription: String {get} //NOT related to event content (e.g. workshop content)
+    var color : String {get}
+    var hypePeriod : DateInterval {get}
+    var imageURL : String? {get}
+    var endTime : Date? {get}
+    var venue : String? {get} //for room names etc.
+    var facebookLink : String? {get}
```

Comment:
What do we think about making this an ID instead of a URL? Allows us to generate the URL on the fly and potentially use the FB API if we want to.

Reply:
As in this would be specifically the facebook event id? I'd be all for that :)